Assets/MirageXR/Player/Scripts/Augmentations/Model/Model.cs

```diff
@@ -24,17 +23,12 @@ private void Start()
 
         private void OnDestroy()
         {
-            UnSubscribe();
-        }
-
-        private void OnEnable()
-        {
-            EventManager.OnEditModeChanged += EditModeChanges;
-        }
+            if (GetComponentInParent<PoiEditor>())
+            {
+                GetComponentInParent<PoiEditor>().EnableBoundsControl(false);
+            }
```

Comment:
the `GetComponentInParent` operation is not the easiest and you should try to avoid calling it

```suggestion
            var poiEditor = GetComponentInParent<PoiEditor>();
            if (poiEditor )
            {
                poiEditor.EnableBoundsControl(true);
            }
```